puppet-lint-optional_default-check.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |spec|
   spec.name        = 'puppet-lint-optional_default-check'
-  spec.version     = '1.0.0'
+  spec.version     = '1.1.0'
   spec.license     = 'MIT'
   spec.author      = 'Vox Pupuli'
   spec.homepage    = 'https://github.com/voxpupuli/puppet-lint-optional_default-check'
```

Comment:
I think you may need to include `CHANGELOG.md` in `spec.files` below for it to be part of the gem itself.

Reply:
updated it